src/components/Popover/types.ts

```diff
@@ -1,7 +1,6 @@
 import type {ValueOf} from 'type-fest';
 import type {PopoverAnchorPosition} from '@components/Modal/types';
 import type BaseModalProps from '@components/Modal/types';
-import type {WindowDimensionsProps} from '@components/withWindowDimensions/types';
 import type CONST from '@src/CONST';
 
 type AnchorAlignment = {
```

Comment:
```suggestion
type AnchorAlignment = ChildrenProps & {
```

and remove children prom types

Reply:
to remove additional prop and be consistent in all places (if sth will change it will change only ion one place and apply everywhere)